README.md

```diff
@@ -446,6 +424,75 @@ $notice = AdminNotices::show('my_notice', 'This is a notice')
     ->notDismissible();
 ```
 
+## Custom Notices
+
+Sometimes you want to display a notice, but you want to completely style it yourself. This is
+possible and pretty straightforward to do.
+
+Start by using the `custom` method on the notice. This will disable all standard visual and behavior
+
+```php
+use StellarWP\AdminNotices\AdminNotices;
+
+$notice = AdminNotices::show('my_notice_custom', 'This is a notice')
+    ->custom();
+```
+
+### `location()`
+
+***Default:*** 'standard'
+
+Sets the location in one of the standard WordPress notice locations. By default, the notice will be
+displayed in the standard location.
+
+Parameters:
+
+1. `string $location` - The location to display the notice. Can be 'standard', 'above_header', '
+   below_header', or 'inline'. Note that 'standard' and 'below_header' are the same location.
+
+```php
+use StellarWP\AdminNotices\AdminNotices;
+
+// Display the notice above the header
+$notice = AdminNotices::show('my_notice_custom', 'This is a notice')
+    ->custom()
+    ->location('above_header');
+```
+
+### Dismissing
+
+The `dismissible` method is not available for custom notices. If you want to add a dismiss button,
+you will need to do so manually. Fortunately, there is a simple way to do this.
+
+```php
+use StellarWP\AdminNotices\AdminNotices;
+use StellarWP\AdminNotices\AdminNotice;
+use \StellarWP\AdminNotices\DataTransferObjects\NoticeElementProperties;
+
+$renderCallback = function (AdminNotice $notice, NoticeElementProperties $elements) {
+    return "
+        <div>
+            <p>This is a custom notice</p>
+            <button type='button' {$elements->customCloserAttributes()}>
```

Comment:
I can't help but feel like `$elements->customCloserAttributes()` is a non-intuitive name.

Perhaps we can provide an alias method like `$elements->enableDismiss()` or `$elements->closeAttributes()` or something similar. I think maybe I'm just stumbling over the words `customCloser`.

Reply:
I can appreciate the weird wording. The "custom" prefix was intended to indicate that this is really only useful for custom notices. And the "closer" part is to indicate that it's used on the thing which will trigger the close.

That said, I have no issues with `$elements->closeAttributes()` if that feels more intuitive to you! 😃

Reply:
Resolved in 8c8af8f